lib/x11utils.pm

```diff
@@ -609,6 +610,19 @@ sub x11_start_program_xterm {
     }
 }
 
+=head2 x11_default_test_terminal {
+
+    x11_default_test_terminal()
+
+Returns the default console to be used by test modules, defaults to xterm
+
+=cut
+
+sub x11_default_test_terminal {
+    return "gnome-terminal" if is_sle("<16") || is_leap("<16");
```

Comment:
That should only be valid on GNOME tests on those platforms (not sure if we pass here in an Leap/KDE test runs, but there gnome-terminal would be wrong)